wagtail/admin/templatetags/wagtailuserbar.py

```diff
@@ -79,10 +81,14 @@ def wagtailuserbar(context, position="bottom-right"):
                     ExplorePageItem(page),
                     EditPageItem(page),
                     AddPageItem(page),
+                    AccessibilityItem(),
                 ]
         else:
             # Not a page.
-            items = [AdminItem()]
+            items = [
+                AdminItem(),
+                AccessibilityItem(),
+            ]
```

Comment:
Looking good. I was expecting us to want to update tests for this but we don’t seem to have any in place for the similar `AdminItem`. So will defer adding tests until we can discuss with @laymonage which ones to add.